utils/historyEvent.ts

```diff
@@ -1,3 +1,5 @@
+import formatBalance from '@/utils/formatBalance'
+
 export const enum HistoryEventType {
```

Comment:
Can we use that one from minimark?

Reply:
Because of the `interaction` from minimark lack of UNLIST event.

We need to add `UNLIST` to `minimark` first, so that I could use it.

Reply:
so far this could be a hack?

Reply:
https://stackoverflow.com/questions/52857698/extending-enum-in-typescript

Reply:
OK, I will modify it later